crates/typst-syntax/src/parser.rs

```diff
@@ -1786,3 +1791,41 @@ impl<'s> Parser<'s> {
         self.nodes.drain(start..end);
     }
 }
+
```

Comment:
Can you please either uncomment this code or fix the tests? :)

Reply:
This code should probably deleted since we don't use unit tests for the parser.